src/Structs/Meta/Article.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace romanzipp\Seo\Structs\Meta;
+
+use romanzipp\Seo\Structs\Meta;
+use romanzipp\Seo\Structs\Struct;
+
+/**
+ * @see https://github.com/joshbuchea/HEAD#facebook-open-graph
+ */
+class Article extends Meta
+{
+    protected $unique = true;
+
+    protected $uniqueAttributes = ['property'];
+
+    public function property($value = null, bool $escape = true): Struct
+    {
+        $this->addAttribute('property', 'article:' . $value, $escape);
+
+        return $this;
+    }
+
+    public function setUnique(bool $unique = true): Struct
+    {
+        $this->unique = $unique;
+
+        return $this;
+    }
```

Comment:
I would rather move this method the `romanzipp\Seo\Structs\Struct` class

Reply:
Sure, no problem 👍️

Reply:
Done @romanzipp